src/main/webapp/i18n/de/aboutUs.json

```
@@ -12,6 +12,23 @@
                 "uploadExercises": "Übungen mit Hochladen von Dateien mit manueller Bewertung"
             }
         },
+        "exam": {
+            "title": "Artemis unterstützt folgende Features für Klausuren",
+            "detail": {
+                "supportedExerciseTypes": "Alle Aufgabentypen sind auch in der Klausur verfügbar",
+                "randomSelectionOfVariants": "Mehrere Varianten von Aufgaben können erstellt und zufällig den Studenten zugewiesen werden",
+                "fullFeatureOverview": {
+                    "description": "Gesamter Überblick über alle Features:",
+                    "studentLink": "Überblick für Studenten",
+                    "instructorLink": "Überblick für Instructors"
+                },
+                "instructions": {
+                    "description": "Anleitung für das Klausurfeature:",
+                    "studentLink": "Anleitung für Studenten",
+                    "instructorLink": "Anleitung für Instructors"
```

```suggestion
                    "instructorLink": "Überblick für Dozenten"
                },
                "instructions": {
                    "description": "Anleitung für das Klausurfeature:",
                    "studentLink": "Anleitung für Studenten",
                    "instructorLink": "Anleitung für Dozenten"
```
Instructors are called "Dozenten" in other german translation strings, this should be consistent.